projects/igniteui-angular/src/lib/grids/common/cell.interface.ts

```diff
@@ -0,0 +1,13 @@
+import { ColumnType } from './column.interface';
+import { RowType } from './row.interface';
+
+export interface CellType {
+    gridID: string;
+    column: ColumnType;
+    columnIndex: number;
+    row: RowType;
+    rowData: any;
+    rowIndex: number;
+    value: any;
+    visibleColumnIndex: number;
+}
```

Comment:
Don't you think it is good idea the column to provide all the information about the column like visibleColumnIndex, column index, field and etc. , same applies to the row?!

Reply:
We will stick to `IgxCell` and get rid of `CellType`